lib/util/createDomain.js

```diff
@@ -1,13 +1,14 @@
 "use strict";
 const url = require("url");
+const getLocalIP = require("./getLocalIP");
 
 module.exports = function createDomain(options) {
 	const protocol = options.https ? "https" : "http";
 
 	// the formatted domain (url without path) of the webpack server
 	return options.public ? `${protocol}://${options.public}` : url.format({
 		protocol: protocol,
-		hostname: options.host,
+		hostname: options.lan ? getLocalIP() : options.host,
```

Comment:
@shellscape  You're right, I will change the word to `useLocalIp` to make it more semantic.
And thank you for giving me all the advice.